subsys/bluetooth/host/scan.c

```diff
@@ -968,13 +976,14 @@ void bt_hci_le_per_adv_sync_established(struct net_buf *buf)
 	if (atomic_test_bit(pending_per_adv_sync->flags,
 			    BT_PER_ADV_SYNC_SYNCING_USE_LIST)) {
 		/* Now we know which address and SID we synchronized to. */
```

Comment:
Update comment?

Reply:
IMO this comment is still valid, and just because we are copying the address in a different way, doesn't change it.

Reply:
You're right.